css/src/components/notification.scss

```diff
@@ -58,6 +62,7 @@ $notification-border-radius: $border-radius !default;
 	a:not(.button) {
 		color: currentColor;
 		font-weight: $weight-semibold;
+		line-height: 1.15;
```

Comment:
should `line-height` be set to normal here?

Reply:
Removed, and set up `line-height` on the component's level